components/sdmmc/sdmmc_cmd.c

```diff
@@ -70,6 +74,7 @@ static esp_err_t sdmmc_read_sectors_dma(sdmmc_card_t* card, void* dst,
         size_t start_block, size_t block_count);
 
 
+#include "soc/sdmmc_struct.h"
```

Comment:
It would be best to keep low-level driver related things out of the high level part of the driver.